### PR TITLE
HPCC 14999 preprocessor: replace USE_XXX with _USE_XXX 

### PR DESCRIPTION
**Pull request by xwang2713**
change "USE_XALAN" and "USE_OPENSSL" to "_USE_XALAN" and "_USE_OPENSSL" in C++ code.
Also remove USE_TBB sections in ecl/hql/hqlexpr.ipp and ecl/hql/hqltrans.ipp per discussion with @ghalliday 

Signed-off-by: Xiaoming Wang xiaoming.wang@lexisnexi.com

### CODE REVIEW

**Comment by richardkchapman** [comment]
The merge should not be in this branch - please rebase

File: esp/src/Visualization [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit ae58f3281e6c48d30866cf7664db468b3f12cc14
+Subproject commit a40abcd1b85efe2802f5c5cba1c3fa529ce01af3
```
Comment (richardkchapman) [inline esp/src/Visualization]: This change is unrelated

**Comment by richardkchapman** [comment]
Changes look ok but the branch has an unwanted merge commit and an unwanted submodule change in it

File: ecl/hql/hqlexpr.ipp [hunk 2]
```diff
@@ -40,7 +40,7 @@
 #include "defvalue.hpp"
 #include "hqlexpr.hpp"
 
-#ifdef USE_TBB
+#ifdef _USE_TBB
```
Comment (ghalliday) [inline ecl/hql/hqlexpr.ipp]: This #if .. #endif should also be deleted.

File: ecl/hql/hqltrans.ipp [hunk 3]
```diff
@@ -30,7 +30,7 @@
 
 #include "hqlexpr.hpp"
 #include "jset.hpp"
-#ifdef USE_TBB
+#ifdef _USE_TBB
```
Comment (ghalliday) [inline ecl/hql/hqltrans.ipp]: Should be deleted rather than changed.

**Comment by ghalliday** [comment]
@xwang2713 See comments above

**Comment by xwang2713** [comment]
I will create a new branch and submit pull request.

**Comment by xwang2713** [comment]
new pull request #8296
